src/utils.rs

```diff
@@ -404,28 +411,56 @@ pub(crate) async fn launch(
     println!("{} was selected", exe.display());
 
     #[cfg(not(target_os = "windows"))]
-    let should_use_wine = os == &BuildOs::Windows;
-    let (binary, args) = (
-        #[cfg(target_os = "windows")]
-        exe.to_str().unwrap().to_owned(),
-        #[cfg(not(target_os = "windows"))]
-        if should_use_wine {
-            wine_bin.unwrap().to_str().unwrap().to_owned()
+    let should_use_wine = (os == &BuildOs::Windows) && !no_wine;
+    let wrapper_string = if wrapper.is_some() {
+            wrapper.unwrap_or_default().to_str().unwrap().to_owned()
+        } else {
+            "".to_owned()
+        };
+
+    let wrapper_vec = if !wrapper_string.is_empty() {
+        split(&wrapper_string.to_owned()).unwrap()
+    } else {
+        Vec::<String>::new()
+    };
```

Comment:
You can use shadow variables here. I think you can just reuse the `wrapper` variable name from here on down